example/spec/helpers/testUtils.js

```diff
@@ -137,7 +137,7 @@ const loadConfig = async (type = 'app') => {
   await loadEnvFile('./.env');
   verifyRequiredEnvironmentVariables();
 
-  const configFromFile = process.env.KES_DEPLOYMENT.toString() === 'true' ?
+  const configFromFile = process.env.KES_DEPLOYMENT === 'true' ?
```

Comment:
This might cause confusion if it is ever set to `true` (the boolean value) somehow, though. What about
```suggestion
  const configFromFile = `${process.env.KES_DEPLOYMENT}` === 'true' ?
```

Reply:
Environment variables can only ever be strings.

Reply:
The node docs make this sort of ambiguous:

"Assigning a property on process.env will implicitly convert the value to a string. This behavior is deprecated. Future versions of Node.js may throw an error when the value is not a string, number, or boolean."

Reply:
Resolving since `process.env` can apparently only ever contain strings in node.js.